Basic-Car-Maintenance/Shared/BasicCarMaintenanceApp.swift

```diff
@@ -7,16 +7,27 @@
 
 import FirebaseCore
 import SwiftUI
+import SwiftData
 
 @main
 struct BasicCarMaintenanceApp: App {
     @State private var actionService = ActionService.shared
     @UIApplicationDelegateAdaptor(AppDelegate.self) var delegate
+    let container: ModelContainer
+    
+    init() {
+        do {
+            self.container = try ModelContainer(for: AcknowledgedAlert.self)
+        } catch {
+            fatalError("Failed to create ModelContainer for AcknowledgedAlert.")
```

Comment:
Is it likely this ever fails? Because this. would mean the whole app can't ever be initialized if the `container` fails to initialize

Reply:
Will we later be needing to do a database migration here? (whenever it comes to that)

Reply:
So far I don't think we will because only the id is all what we want to control the alerts

Reply:
I agree, I think at the moment the `id` is fine